rskj-core/build.gradle

```diff
@@ -36,16 +36,16 @@ repositories {
     }
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 8
```

Comment:
shouldn't be `1.8` ?

Reply:
🤔 good point, the question is why it is working under a local java 8... maybe it is smart enough to know it is 1.8? I could not find anything related in doc, so will change it just in case.